libs/sdk-ui-filters/src/AttributeFilter@next/AttributeFilterBase.tsx

```diff
@@ -1,82 +1,304 @@
+     *
+     * Note: It's not possible to combine this property with "connectToPlaceholder" property. Either - provide a value, or a placeholder.
+     * The 'onApply' callback must be specified in order to handle filter changes.
+     */
+    filter?: IAttributeFilter;
+
+    /**
+     * Specifies a parent attribute filter that will be used to reduce options for for current attribute filter.
+     *
+     * @remarks
+     * Parent filters elements must contain their URIs due to current backend limitations.
+     */
+    parentFilters?: AttributeFiltersOrPlaceholders;
+
+    /**
+     * Specify {@link @gooddata/sdk-ui#IPlaceholder} to use to get and set the value of the attribute filter.
+     *
+     * @remarks
+     * Note: It's not possible to combine this property with "filter" property. Either - provide a value, or a placeholder.
+     * There is no need to specify 'onApply' callback if 'connectToPlaceholder' property is used as the value of the filter
+     * is set via this placeholder.
+     */
+    connectToPlaceholder?: IPlaceholder<IAttributeFilter>;
+
+    /**
+     * Specify the over attribute - an attribute the filter and its parent filter are connected through.
+     *
+     * @remarks
+     * You can either provide an {@link @gooddata/sdk-model#ObjRef} which will be used for all the parent filters,
+     * or you can provide a pure function that will be called for each parent filter to determine the respective over attribute.
+     */
+    parentFilterOverAttribute?: ParentFilterOverAttributeType;
+
+    /**
+     * Specify identifier of attribute display form, for which you want to construct the filter.
+     *
+     * @remarks
+     * Note: this is optional and deprecated. If you do not specify this, then you MUST specify the 'filter' prop or 'connectToPlaceholder' prop.
+     *
+     * @deprecated - use the filter prop instead
+     */
+    identifier?: string;
+
+    /**
+     * Specify title for the attribute filter.
+     *
+     * @remarks
+     * By default, the attribute name will be used.
+     */
+    title?: string;
+
+    /**
+     * Locale to use for localization of appearing texts.
+     */
+    locale?: string;
+    /**
+     * Specify function which will be called when user clicks 'Apply' button.
+     *
+     * @remarks
+     * The function will receive the current specification of the filter, as it was updated by the user.
+     *
+     * @param filter - new value of the filter.
+     */
+    onApply?: OnApplyCallbackType;
+
+    /**
+     * Customize attribute filter with a callback function to trigger when an error occurs while
+     * loading attribute elements.
+     */
+    onError?: (error: any) => void;
+
+    /**
+     * Customize attribute filter with a component to be rendered if attribute elements loading fails
+     */
+    ErrorComponent?: React.ComponentType<IAttributeFilterErrorProps>;
```

Comment:
I will split IAttributeFilterBaseProps interface into two. 

One interface could IAttributeFilterBaseCustomization and its content could be list customisable components and final  IAttributeFilterBase could be composition of this two interfaces